Games/core/Meeting.js

```diff
@@ -108,7 +109,7 @@ module.exports = class Meeting {
       this.game.setup.whispers &&
       this.name != "Pregame" &&
       // disable whispers for anonymous meetings that are not the village meeting
-      !(this.anonymous && this.name != "Village")
+      !(this.anonymous && this.name != "Village") && member.alive
```

Comment:
i already added this fix, specifically i made crier, spy etc all abilities not available